BUILD.gn

```diff
@@ -54,55 +54,40 @@ config("branding") {
 }
 
 npm_action("atom_browserify_sandbox") {
-  deps = [
-    ":atom_js2c_copy",
-  ]
+  script = "browserify"
 
-  sandbox_args = [
-    "lib/sandboxed_renderer/init.js",
-    "-r",
-    "./lib/sandboxed_renderer/api/exports/electron.js:electron",
-    "-t",
-    "aliasify",
+  outputs = [
+    "$target_gen_dir/js2c/preload_bundle.js",
   ]
 
-  inputs = [
+  args = [
```

Comment:
why'd `inputs` disappear here? it's necessary to tell GN when it needs to rebuild.